frontend/src/metabase/dashboard/actions/tabs.ts

```diff
@@ -309,6 +318,24 @@ export const tabsReducer = createReducer<DashboardState>(
         }
         const dashCard = state.dashcards[dashCardId];
 
+        if (!dashCard.dashboard_tab_id) {
+          throw Error(
```

Comment:
We always use `throw new Error`. Even though it's [equivalent](https://es5.github.io/#x15.11.1), I'd go for consistency (applies to entire file).